src/pages/TutorProfilePage.tsx

```diff
@@ -0,0 +1,766 @@
+import { headers } from '../util';
+//import { Cloudinary } from '@cloudinary/url-gen';
+
+const TutorProfilePage: React.FC<TutorProfilePageProps> = (isUpdate) => {
+    const { firstName, lastName, email } = JSON.parse(localStorage.getItem('userData') ?? '');
+    const [selectedImage, setSelectedImage] = useState<Blob | null>(null);
+    const [initialValues, setInitialValues] = useState<TutorRequest>({
+        grades: [],
+        about: '',
+        education: '',
+        avatar: '',
+        availability: ['Monday', 'Blanday'],
+        MathSubject: [],
+        ForeignLanguages: [],
+        English: [],
+        SocialStudies: [],
+        Science: [],
+        yearsOfExperience: 0,
+        _id: '',
+        userId: '',
+        __v: 0,
+    });
+    //  const cld = new Cloudinary({ cloud: { cloudName: 'dcgkkskk5' } });
+
+    // interface InitialTutorValues extends TutorRequest {
+    //     _id: string;
+    //     userId: string;
+    // }
+    const tutorData: TutorRequest = {
+        //MOCK DATA FOR A MEANWHILE//
+        availability: [
+            'Monday',
+            'Tuesday',
+            'Wednesday',
+            'Thursday',
+            'Friday',
+            'Saturday',
+            'Sunday',
+        ],
+        about: '',
+        grades: ['K', '1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12'],
+        avatar: '',
+        education: '',
+        yearsOfExperience: 0,
+        MathSubject: [
+            'Math',
+            'Algebra',
+            'Geometry',
+            'Trigonometry',
+            'Calculus',
+            'Statistics',
+            'Pre-Calculus',
+            'SAT Math Test Prep',
+            'ACT Math Test Prep',
+        ],
+        ForeignLanguages: ['Spanish', 'French', 'Chineese', 'German', 'Latin'],
+        English: [
+            'Writing',
+            'Reading',
+            'ESL',
+            'Poetry',
+            'Literacy',
+            'ACT English Test Prep',
+            'ACT Reading Test Prep',
+            'ACT Writing Test Prep',
+        ],
+        SocialStudies: [
+            'World History',
+            'Psychology',
+            'US Government',
+            'Social Science',
+            'US History',
+            'Political Science',
+            'Geography',
+            'European History',
+        ],
+        Science: [
+            'Biology',
+            'Chemistry',
+            'Earth Science',
+            'Physics',
+            'Science',
+            'ACT Science Test Prep',
+        ],
+        _id: '',
+        userId: '',
+        __v: 0,
+    };
+
+    //FETCHING TUTOR DATA FOR AN UPDATE
+    useEffect(() => {
+        if (!isUpdate) {
+            return;
+        }
+        const fetchTutor = async () => {
+            try {
+                const tutorId = localStorage.getItem('tutorId');
+                const response = await axios.get(
+                    `https://ffprac-team2-back.onrender.com/api/v1/tutors/${tutorId}`
```

Comment:
For best practices I would recommend to hide the full URL used for making requests in the .env file. This guarantees a more secure management of confidential data such as API URLs and helps the process of changing URLs without modifying the application code. I would use something like REACT_APP_BASE_URL for instance.